src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -132,54 +132,38 @@
             <tag name="api_platform.filter" />
         </service>
 
-<!--        <service id="sylius.api.order_date_filter" parent="api_platform.doctrine.orm.date_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="checkoutCompletedAt">exclude_null</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
-
-<!--        <service id="sylius.api.order_product_filter" parent="api_platform.doctrine.orm.search_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="items.productName">exact</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
-
-<!--        <service id="sylius.api.order_shipping_method_filter" parent="api_platform.doctrine.orm.search_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="shipments.method.code">exact</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
-
-<!--        <service id="sylius.api.order_variants_filter" parent="api_platform.doctrine.orm.search_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="items.variant.translations.name">exact</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
+        <service id="sylius_api.admin.search_filter.order" parent="api_platform.doctrine.orm.search_filter" public="true">
+            <argument type="collection">
+                <argument key="channel.code" />
+                <argument key="currencyCode">exact</argument>
+                <argument key="customer.id">exact</argument>
+                <argument key="items.productName">exact</argument>
+                <argument key="shipments.method.code">exact</argument>
+                <argument key="items.variant.translations.name">exact</argument>
```

Comment:
Not related; why do we check the current variant name, but a potentially snapshotted product name?

Reply:
I don't know